src/modular/bingcd/xgcd.rs

```diff
@@ -163,11 +160,13 @@ impl<const LIMBS: usize> OddUint<LIMBS> {
     ///
     /// The function executes in time variable in `iterations`.
     #[inline]
-    pub(super) const fn partial_binxgcd_vartime<const UPDATE_LIMBS: usize>(
+    pub(super) const fn partial_binxgcd_vartime<
```

Comment:
I think this could be renamed `partial_binxgcd`. As mentioned the runtime is determined by `iterations`, it doesn't short-circuit.